src/App.test.js

```diff
@@ -1,9 +1,8 @@
 import React from "react";
-import { render, screen } from "@testing-library/react";
+import { render } from "@testing-library/react";
 import App from "./App";
 
 test("example test", () => {
   render(<App />);
-  const linkElement = screen.getByText("Meine Lieblingsübungen");
```

Comment:
Seemed not to be useful to check this...
Should I delete the whole file?

Reply:
I will :)